src/main/java/edu/greenblitz/robotName/commands/arm/elbow/MoveElbowToAngle.java

```diff
@@ -18,14 +21,36 @@ public MoveElbowToAngle(ElbowConstants.PresetPositions targetAngle) {
 
     @Override
     public void initialize() {
-        elbow.moveToAngle(targetAngle);
+        isElbowCrashableIntoWall = SwerveChassis.getInstance().isRobotNearBoundsOfField();
```

Comment:
We need to add driver indication for this, if the driver tries to deploy the arm but nothing is happening we need to tell her why it is happening

Reply:
It will be in the driver dashboard.

Reply:
Sababa, also talk to Noam about LED indicator, not in this PR